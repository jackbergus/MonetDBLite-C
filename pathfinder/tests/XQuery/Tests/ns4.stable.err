stderr of test 'ns4` in directory 'tests/XQuery` itself:


# 09:47:26 >  
# 09:47:26 >  Mtimeout -timeout 180 Mserver "--config=/opt/MonetDB/etc/MonetDB.conf" --debug=10 --set "monet_mod_path=/home/teubner/pf/untouched/runtime/.libs:/opt/MonetDB/lib/MonetDB" --set "gdk_dbfarm=/home/teubner/pf/untouched/dbfarm" --set "sql_logdir=/home/teubner/pf/untouched/log" --set mapi_port=32832 --set sql_port=48274 --set xquery_port=51704 --set monet_prompt= --trace "--dbname=mTests_tests_XQuery" --dbinit="module(pathfinder); pfstart();" ; echo ; echo Over..
# 09:47:26 >  


# 09:47:26 >  
# 09:47:26 >  Mtimeout -timeout 60 MapiClient -lxquery -sxml -u monetdb -P monetdb --host=localhost --port=51704 ns4.xq
# 09:47:26 >  


# 09:47:26 >  
# 09:47:26 >  Done.
# 09:47:26 >  

