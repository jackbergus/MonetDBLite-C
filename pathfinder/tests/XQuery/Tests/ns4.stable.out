stdout of test 'ns4` in directory 'tests/XQuery` itself:


# 09:47:26 >  
# 09:47:26 >  Mtimeout -timeout 180 Mserver "--config=/opt/MonetDB/etc/MonetDB.conf" --debug=10 --set "monet_mod_path=/home/teubner/pf/untouched/runtime/.libs:/opt/MonetDB/lib/MonetDB" --set "gdk_dbfarm=/home/teubner/pf/untouched/dbfarm" --set "sql_logdir=/home/teubner/pf/untouched/log" --set mapi_port=32832 --set sql_port=48274 --set xquery_port=51704 --set monet_prompt= --trace "--dbname=mTests_tests_XQuery" --dbinit="module(pathfinder); pfstart();" ; echo ; echo Over..
# 09:47:26 >  

# Monet Database Server V4.9.0
# Copyright (c) 1993-2005, CWI. All rights reserved.
# Compiled for i686-pc-linux-gnu/32bit with 32bit OIDs; dynamically linked.
# Visit http://monetdb.cwi.nl/ for further information.

printf("\nReady.\n");

Ready.
quit();

Over..

# 09:47:26 >  
# 09:47:26 >  Mtimeout -timeout 60 MapiClient -lxquery -sxml -u monetdb -P monetdb --host=localhost --port=51704 ns4.xq
# 09:47:26 >  

<?xml version="1.0" encoding="utf-8"?>
<XQueryResult><a:bar xmlns:a="http://asdf.com/example1"/><b:bar xmlns:b="http://asdf.com/example2"/></XQueryResult>

# 09:47:26 >  
# 09:47:26 >  Done.
# 09:47:26 >  

